enzyme/Enzyme/CMakeLists.txt

```diff
@@ -8,8 +8,10 @@ endif()
 
 get_target_property(TBL_LINKED_LIBS LLVMSupport INTERFACE_LINK_LIBRARIES)
 if (NOT TBL_LINKED_LIBS)
+message(STATUS "No TBL_LINKED_LIBS found")
 else()
-list(REMOVE_ITEM TBL_LINKED_LIBS "ZLIB::ZLIB")
+message(STATUS "TBL_LINKED_LIBS (test): ${TBL_LINKED_LIBS}")
```

Comment:
I'm fairly certain this was intentionally removed due to a bug in LLVM's packaging of the dylib

Reply:
How do you work around it, since linking failed on the Julia side without this line.
Also, any other requests before landing this?

Reply:
let's do these things one at a time to confirm they're fine.

Can you leave this as a non-functional change (aka keep old zlib behavior) first, but add the non dylib linking, and then a follow up for zlib change?